package.json

```
@@ -32,12 +32,12 @@
     "eslint-plugin-regexp": "^1.15.0"
   },
   "devDependencies": {
-    "@stylelint/prettier-config": "^2.0.0",
-    "eslint": "^8.42.0",
-    "eslint-plugin-jest": "^27.2.1",
-    "jest": "^29.5.0",
+    "@stylelint/prettier-config": "github:stylelint/prettier-config#prettier-v3",
```

```suggestion
    "@stylelint/prettier-config": "^3.0.0",
```

Commit suggestion for when we merge https://github.com/stylelint/prettier-config/pull/22 and release `3.0.0`.

(It's just a coincidence that the package numbers match.)

---

Thanks. Fixed! 97f4ef8